task_11/style.css

```diff
@@ -0,0 +1,90 @@
+/* CSS Document */
+.header {
+    position: fixed;
+    left: 0px;
+    right: 0px;
+    top: 0px;
+    height: 50px;
+    line-height: 50px;
+    border-bottom: 1px solid grey;
+}
+
+.body {
+    margin: 0;
+}
+.main {
```

Comment:
Why main and main_header classes are contain the same styles?)

Reply:
Done.